frontend/packages/kubevirt-plugin/src/statuses/vm/vm-status.ts

```diff
@@ -193,13 +196,20 @@ const isBeingImported = (
   return null;
 };
 
-const isBeingStopped = (vm: VMKind, vmi: VMIKind = null): VMStatusBundle => {
-  const vmiPhase = getStatusPhase<VMIPhase>(vmi);
-  // according to https://github.com/kubevirt/kubevirt/blob/master/pkg/virt-api/rest/subresource.go
-  const isFinalPhase =
-    !vmiPhase || [VMIPhase.Unknown, VMIPhase.Failed, VMIPhase.Succeeded].includes(vmiPhase);
+const isVMError = (vm: VMKind): VMStatusBundle => {
```

Comment:
Nitpick that doesn't need to be addressed now. 

Typically, when I see a function that starts with 'is', I expect a boolean return type. I think we should eventually change these function names to better match what they're doing, such as `getStoppingStatus`. If we need boolean tests for them, it would be trivial to create functions that coerce the return values of these functions to a boolean. Just my two cents.

Reply:
Yes I agree. I think we should refactor the whole logic of this file to make more sense. And not be it one dimensional  array of ifs. And test it as well.

Nevertheless, I think we should not do it here in this bug fix but a bit later.